cg/store/api/findbusinessdata.py

```diff
@@ -317,6 +318,20 @@ def pool(self, pool_id: int) -> models.Pool:
         """Fetch a pool."""
         return self.Pool.get(pool_id)
 
+    def ready_made_library_expected_reads(self, case_id: str) -> float:

```

Comment:
Can the expected reads be a float? Why not an int?

Reply:
Thought it would be a float if a decimal was created, I was wrong :)